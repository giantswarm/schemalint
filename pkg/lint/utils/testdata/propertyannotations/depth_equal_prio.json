{
    "$defs": {
        "foo": {
            "properties": {
                "childProp": {
                    "type": "object",
                    "properties": {
                        "grandchildProp": {
                            "type": "string",
                            "title": "Grandchild Ref Prop"
                        }
                    }
                }
            },
            "type": "object"
        },
        "bar": {
            "properties": {
                "grandchildProp": {
                    "type": "string",
                    "title": "Grandchild Ref Ref Prop"
                }
            }
        }
    },
    "properties": {
        "rootProp": {
            "$ref": "#/$defs/foo",
            "properties": {
                "childProp": {
                    "$ref": "#/$defs/bar"
                }
            },
            "type": "object"
        }
    },
    "type": "object"
}
