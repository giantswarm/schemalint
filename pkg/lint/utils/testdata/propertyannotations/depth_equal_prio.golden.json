{
  "/properties/rootProp": {
    "Title": null,
    "Description": null,
    "Examples": null
  },
  "/properties/rootProp/properties/childProp": {
    "Title": null,
    "Description": null,
    "Examples": null
  },
  "/properties/rootProp/properties/childProp/properties/grandchildProp": {
    "Title": {
      "Value": "Grandchild Ref Ref Prop",
      "ReferenceLevel": 1
    },
    "Description": null,
    "Examples": null
  }
}
